cmd/list_test.go

```diff
@@ -110,3 +110,90 @@ func TestWhoCan_ValidateNamespace(t *testing.T) {
 	})
 
 }
+
+func TestWhoCan_policyMatches(t *testing.T) {
+
+	data := []struct {
+		name string
+
+		verb         string
+		resource     string
+		resourceName string
+
+		rule rbacv1.PolicyRule
+
+		matches bool
+	}{
+		{
+			name: "scenario1",
```

Comment:
Does the command work for singular and plural forms? For example "get pods", "get pod", "get pods foo" and "get pod foo". If it works already, please could you add some test cases to cover those options? If it needs extra work  an issue / PR would be fine.

Reply:
Nope. Currently we only use stings comparison so `get pod foo` won't work.  I'd take it as a dedicated issue / PR to add also checks for resource existence, e.g.:

```
./kubectl-who-can get frogs
Warning: the server doesn't have a resource type 'frog'
```

BTW  I realized that`kubectl auth can-i` supports wildcards:
```
kubectl auth can-i '*' pods --as=dpacak
```

Do you think it makes sens to add it to who-can as well?

Reply:
Yes to both!